### PR TITLE
Fixes russian translate

### PR DESCRIPTION
**Pull request by AlexTriada**

### CODE REVIEW

File: A3-Antistasi/Stringtable.xml [hunk 1]
```diff
@@ -306,12 +307,14 @@
         <Russian>Нет</Russian>
       </Key>
       <Key ID="STR_antistasi_dialogs_generic_button_yes_tooltip">
-        <Original />
-        <English />
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: This is the correct way to format empty XML containers.

File: A3-Antistasi/Stringtable.xml [hunk 2]
```diff
@@ -337,6 +340,7 @@
       </Key>
       <Key ID="STR_antistasi_credits_generic_version_text">
         <Original>2.1.1</Original>
+        <English>2.1.1</English>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: These lines aren't required. The number will be the same throughout.

File: A3-Antistasi/Stringtable.xml [hunk 3]
```diff
@@ -337,6 +340,7 @@
...
         <Russian>2.1.1</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: These lines aren't required. The number will be the same throughout.

File: A3-Antistasi/Stringtable.xml (hunk 1 above)
Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml]: OK)

Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml]: Can delete them at all? Since they are empty?

Comment (Toakan) [reply A3-Antistasi/Stringtable.xml]: `<Original />` needs to be there.

**Comment by Bob-Murphy** [comment]
Checked and merged.

**Pull request merged**